README.md

```diff
@@ -240,7 +240,7 @@ General options and their default values:
     ENABLE_SYSTEM_GMP=ON       - Use GMP from system (much faster than bundled mini-gmp)
     ENABLE_SYSTEM_JSONCPP=OFF  - Use JsonCPP from system
     OPENGL_GL_PREFERENCE=LEGACY - Linux client build only; See CMake Policy CMP0072 for reference
-    RUN_IN_PLACE=FALSE         - Create a portable install (worlds, settings etc. in current directory)
+    UPDATE_CHECKER_DEFAULT=TRUE - Default behavior of the version update checker
```

Comment:
What does true/false mean?  Perhaps the setting should be `ENABLE_UPDATE_CHECKER`